doc/user/content/ingest-data/warpstream.md

```diff
@@ -0,0 +1,138 @@
+
+#### Before you begin
+
+Ensure you have the following:
+
+-   WarpStream account: [Register here](https://console.warpstream.com/signup)
+-   Fly.io account: Used for deploying a WarpStream cluster with TLS termination and SASL authentication.
+
+1. #### Set up WarpStream
+
+    If you already have a WarpStream cluster, you can skip this step.
+
+    a. **Sign Up**: Begin by registering for a WarpStream account.
+
+    b. **Deploy on Fly.io**: Follow [this guide](https://github.com/warpstreamlabs/warpstream-fly-io-template) to deploy your WarpStream cluster on Fly.io.
+
+    c. **Generate Credentials**: Post deployment, [create credentials](https://docs.warpstream.com/warpstream/how-to/configure-the-warpstream-agent-for-production/configure-authentication-for-the-warpstream-agent#sasl-authentication) for connecting to your WarpStream cluster.
+
+    d. **Test Connection**: Use the provided credentials to connect to the WarpStream cluster on Fly.io. Test this connection using [the WarpStream CLI](https://docs.warpstream.com/warpstream/install-the-warpstream-agent):
+
+    ```bash
+    warpstream kcmd -type diagnose-connection \
+                    -bootstrap-host $CLUSTER_NAME.fly.dev \
+                    -tls -username ccun_XXXXXXXXXX \
+                    -password ccp_XXXXXXXXXX
+    ```
+
+    Change the `bootstrap-host` to the name of your WarpStream cluster on Fly.io.
+
+    e. **Topic Creation**: Establish the `materialize_click_streams` topic:
+
+    ```bash
+    warpstream kcmd -bootstrap-host $CLUSTER_NAME.fly.dev \
+                    -tls -username ccun_XXXXXXXXX \
+                    -password ccp_XXXXXXXXXX \
+                    -type create-topic \
+                    -topic materialize_click_streams
+    ```
+
+    f. **Produce Sample Records**: Generate and push sample records for testing:
+
+    ```bash
+    warpstream kcmd -bootstrap-host $CLUSTER_NAME.fly.dev \
+                    -tls -username ccun_XXXXXXXXXX \
+                    -password ccp_XXXXXXXXXX \
+                    -type produce \
+                    -topic materialize_click_streams_1 \
+                    --records '{"action": "click", "user_id": "user_0", "page_id": "home"},,{"action": "hover", "user_id": "user_0", "page_id": "home"},,{"action": "scroll", "user_id": "user_0", "page_id": "home"}'
+    ```
+
+    > **Note:** The WarpStream CLI uses `,,` as a delimiter between JSON records.
+
+2. #### Integrate with Materialize
+
+    To integrate WarpStream with Materialize, you need to set up a connection to the WarpStream broker and create a source in Materialize to consume the data.
+
+    Head over to the Materialize console and follow the steps below:
+
+    a. Save WarpStream credentials:
```

Comment:
In the Step 1., each sub-steps starts with a brief description:
```
f. **Produce Sample Records**:
```
But in the Step 2.,  the sub-steps do not have this intro

Reply:
Do you suggest that it is better to remove the brief description from step 1 or to add a brief description to step 2?

Reply:
My vote is to remove the description. I like how it looks but it doesn't match the structure from the other docs.

Reply:
Sounds good! I've removed those descriptions so that it could be consistent with the rest of the docs.